README.md

```diff
@@ -8,13 +8,14 @@ The Publishing & Information Hub will be responsible for the following:
 * Provide functionality to display information in court and tribunals buildings on the relevant hardware
 * Comply with Open Justice procedures and business rules 
 
+
 ## Getting Started
 
 ### Prerequisites
 
 Running the application requires the following tools to be installed in your environment:
 
-  * [Node.js](https://nodejs.org/) v12.0.0 or later
+  * [Node.js](https://nodejs.org/) v12.0.0 to v14.17.1 (later version will not work)
```

Comment:
not sure if we need the `(later version will not work)` i think as our dependencies that use node update, our compatibility with later versions will ease up too, maybe just keep the `v12.0.0 to v14.17.1` for now?